### ISSUE TITLE
Weave.jl and jmd

### ISSUE THREAD

**Issue by rafaqz** [JuliaEditorSupport/julia-vim#182]
It would be great to support julia code blocks in Weave.jl jmd files. Code highlighting inside blocks would of course be great, but simple things like completing latex would be amazing.

**Comment by jessebett** [comment]
In addition to syntax highlighting I would like the ```julia blocks inside `.jmd` files to use the `julia-vim` configuration generally. Including automatically indenting, parenthesis matching, movements, unicode...

**Comment by jessebett** [comment]
For now I have this working by setting the filetype to `markdown` and using the `vim-markdown` syntax highlight fencing.

**Comment by cmcaine** [comment]
It's worth noting that indent support, text objects and help don't work yet.

Text objects and help would be easy to add. Indent support is a bit trickier (I think emacs is better at composing modes ;))

**Comment by cmcaine** [comment]
This should have been closed by https://github.com/JuliaEditorSupport/julia-vim/pull/237. Is it not working for you?

**Comment by newptcai** [comment]
Yes. It actually works. Thanks. Perhaps this should also be closed?
